src/Panel.php

```diff
@@ -188,4 +187,24 @@ public function isEnabled()
     {
         return true;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getViewPath()
```

Comment:
Should be type-hinted but that's something to change in the interface first.

```suggestion
    public function getViewPath(): string
```

Reply:
will be done globally after this PR gets merged.